executor/common_kvm_arm64.h

```diff
@@ -195,6 +196,14 @@ static void setup_cpu_with_opts(int vmfd, int cpufd, const struct kvm_opt* opt,
 	init.features[0] = features;
 	// Use the modified struct kvm_vcpu_init to initialize the virtual CPU.
 	ioctl(cpufd, KVM_ARM_VCPU_INIT, &init);
+        // Make sure that PMUs are initialized if enabled
+        if (opt_count == 1 && (features & pmu_bit)) {
+                struct kvm_device_attr attrs = {
+                    .group = KVM_ARM_VCPU_PMU_V3_CTRL,
+                    .attr = KVM_ARM_VCPU_PMU_V3_INIT
+                };
+                ioctl(cpufd, KVM_SET_DEVICE_ATTR, &attrs);
```

Comment:
Is it possible to uninit it back with an ioctl?
I am worried that we are restricting completeness and all states that the fuzzer can achieve.
Perhaps it's better to pass a separate flag from the fuzzer that will say to initialize PMU or not.

Reply:
I'm not sure how easy it is to uninit it, but I think the flag makes sense since conceptually it's exactly what I'm trying to do. Could you please point me to some part of the code that sets up a flag like that so that I can follow that paradigm?